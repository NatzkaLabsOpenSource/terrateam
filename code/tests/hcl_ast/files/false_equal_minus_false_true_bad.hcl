foo = -false true
---
foo = -false 1
---
foo = -1 1
---
foo = -1 foo
