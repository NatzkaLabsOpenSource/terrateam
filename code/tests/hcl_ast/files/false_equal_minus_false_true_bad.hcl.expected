Expected operation after value, found identifier or literal.
